ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsPruning.scala

```diff
@@ -85,7 +85,7 @@ private[backend] trait StorageBackendTestsPruning extends Matchers with StorageB
       contractId = hashCid("#1"),
       signatory = someParty,
     )
-    val createFilter1 = DbDto.CreateFilter(1L, someTemplateId.toString, "signatory")
+    val createFilter1 = DbDto.CreateFilter(1L, someTemplateId.toString, "party")
```

Comment:
this is about  sure that the create filter and the event tables are aligned in terms of filtering. this does not make the test less covering, and needed because M implementation is not using create filter, but the actual event data